frontend/src/components/Header/Header.jsx

```diff
@@ -1,9 +1,16 @@
-import { Example } from "./Header.styled";
+import { Navigation } from "../Navigation/Navigation";

```

Comment:
import { Navigation } from "./Navigation";

should be enought only one "Navigation" and one "." before "/"